src/guest/vm_builder_qemu.cc

```diff
@@ -599,6 +599,7 @@ void VmBuilderQemu::BuildAafCfg(void) {
 
 bool VmBuilderQemu::BuildVgpuCmd(void) {
     std::string vgpu_type = cfg_.GetValue(kGroupVgpu, kVgpuType);
+    std::string powerCtl = cfg_.GetValue(kGroupService, kServPowerCtrl);
```

Comment:
Do not add these code in BuildVgpuCmd(), create a new function named BuildVinputCmd().